assets/home/home.shared.scss

```diff
@@ -82,6 +82,7 @@
 
 		&__grid {
 			display: flex;
+			width: 100%;
```

Comment:
I wasn't able to test this change. Is there a good way to test this change?

Reply:
I'll send you a custom environment in private. 😉